sdkcompat/v242/com/google/idea/sdkcompat/clion/CMakeNotificationProviderWrapper.java

```diff
@@ -17,32 +17,20 @@
 package com.google.idea.sdkcompat.clion;
 
 
+import com.intellij.openapi.extensions.ExtensionPoint;
 import com.intellij.openapi.fileEditor.FileEditor;
 import com.intellij.openapi.project.Project;
 import com.intellij.openapi.vfs.VirtualFile;
-import com.jetbrains.cidr.cpp.cmake.workspace.CMakeNotificationProvider;
-
-import java.util.function.Function;
 import javax.annotation.Nullable;
-import javax.swing.*;
+import javax.swing.JComponent;
 
-// #api223
+// #api224
```

Comment:
@LeFrosch about the notation - this should be the *last IJ version for which the sdkcompat layer should be kept*. In the other words - when we will decide to drop 223, we will grep for #api223 occurrences and delete/inline everything